tests/test_views.py

```diff
@@ -18,17 +17,20 @@ def test_submit_valid_form(testapp):
     res = form.submit()
     assert res.content_type == 'text/xml'
 
+
 def test_submit_with_shapefile_without_metadata(testapp):
     form = testapp.get('/').form
     form['data'] = Upload('tests/fixtures/SDE_DATA_BD_A8GNS_2003.zip')
     res = form.submit()
     assert res.content_type == 'text/xml'
 
+
 def test_submit_without_shapefile(testapp):
     form = testapp.get('/').form
     res = form.submit()
     assert 'A datafile is required to proceed' in res
 
+
```

Comment:
Yeah, methods are separated by a single space, but module-level functions should have two spaces between them.